src/test/test.js

```diff
@@ -51,7 +51,7 @@ describe('/POST Login user', () => {
      }
   it('it should check if  user exist', (done) => {
     chai.request(app)
-        .post('/api/v1/users/login')
+        .post('/api/v1/auth/login')
```

Comment:
I want to think you spoke with the person who did signin before changing their code.

Reply:
yeah, we did it together